Gemfile.lock

```diff
@@ -29,7 +29,7 @@ GEM
     builder (3.2.3)
     concurrent-ruby (1.1.1)
     crass (1.0.4)
-    double_entry (0.10.1)
+    double_entry (1.0.1)
```

Comment:
### [double_entry](https://github.com/envato/double_entry)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 0.10.1 → 1.0.1

[_~~change-log~~, [source-code](https://github.com/envato/double_entry)_]





<details>
<summary>Commits</summary>

A change of **125** commits. See the full changes on [the compare page](https://github.com/envato/double_entry/compare/v0.10.1...v1.0.1).

These are the first 10 commits:
- (1752d4d) [Run Rubocop in the build](https://github.com/envato/double_entry/commit/1752d4d2ad5ef67cf18a53923f2f42eae06a8b19)
- (4c2b7c9) [Rubocop should fail on error](https://github.com/envato/double_entry/commit/4c2b7c9c1436205c40994c61b1b0d8ce9bc27b42)
- (cda778b) [Create a test to expose the ArgumentError bug in retrieve_ag…](https://github.com/envato/double_entry/commit/cda778b533e79582c2cf1a70d3338d9bde32fe64)
- (2918e12) [Byebug requires Ruby >= 2.0.0](https://github.com/envato/double_entry/commit/2918e12de582d8e64e7440a16881ca0e056af61f)
- (36bb85b) [Use the subject's name](https://github.com/envato/double_entry/commit/36bb85bad79d25baac33c7e6397fed354e2883dc)
- (07ae4e0) [Merge pull request #88 from envato/fix-ArgumentError-in-retr…](https://github.com/envato/double_entry/commit/07ae4e0cd936dedbd366b5a612f243aa3ab475ff)
- (ce2204b) [Patch release version 0.10.2](https://github.com/envato/double_entry/commit/ce2204b41c7e2853063ec1d495c0dd3348bd27ff)
- (808f478) [Specify the version of Rubocop](https://github.com/envato/double_entry/commit/808f47859a17cd0fef3463abdfb2f864886634f9)
- (975d0bc) [new cops that can be bribed till we're compliant](https://github.com/envato/double_entry/commit/975d0bc80d95ea44f6f18f60b710b8def72ca789)
- (3b1186a) [Aggregate spec should call Aggregate directly](https://github.com/envato/double_entry/commit/3b1186a5eeb52b327cee0565d3d61236567c3dc2)


</details>